distbench_netutils.cc

```diff
@@ -107,12 +107,16 @@ absl::StatusOr<DeviceIpAddress> GetBestAddress(std::string_view netdev,
   int score = -1;
   DeviceIpAddress best_match;
   for (const auto& address : all_addresses) {
-    if (address.isLinkLocal() || address.isLoopback()) {
+    if (address.isLinkLocal()) {
       continue;
     }
     int cur_score = (address.isIPv4() == prefer_ipv4);
     // High speed networks in cloudlab are on a private IP range.
     cur_score += address.isPrivate();
+    if (!address.isLoopback()) {
```

Comment:
What's the difference between linklocal and loopback?

Reply:
link local is an ipv6 address "Scope" it allows machines to talk to each other, but is not routable. It is used by the ipv6 replacement for e.g. dhcp.